zh/get-started.md

```diff
@@ -226,607 +228,646 @@ minikube delete
 开始之前，确保以下要求已满足：
 
 - 可以使用 `kubectl` 访问的 Kubernetes 集群
-- [Helm](https://helm.sh/docs/intro/install/): Helm 2 (>= Helm 2.16.5) 或者最新的 Helm 3 稳定版
+- [Helm](https://helm.sh/docs/intro/install/): Helm 2 (>= 2.16.5) 或者最新的 Helm 3 稳定版
 
-1. 安装 TiDB Operator CRDs
+部署 TiDB Operator 的过程分为两步：安装 TiDB Operator CRDs、安装 TiDB Operator。下面分别介绍这两个步骤。
 
-    TiDB Operator 包含许多实现 TiDB 集群不同组件的自定义资源类型 (CRD)。执行以下命令安装 CRD 到集群中：
+### 安装 TiDB Operator CRDs
+
+TiDB Operator 包含许多实现 TiDB 集群不同组件的自定义资源类型 (CRD)。执行以下命令安装 CRD 到集群中：
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.1.6/manifests/crd.yaml
+```
+
+期望输出：
+
+```
+customresourcedefinition.apiextensions.k8s.io/tidbclusters.pingcap.com created
+customresourcedefinition.apiextensions.k8s.io/backups.pingcap.com created
+customresourcedefinition.apiextensions.k8s.io/restores.pingcap.com created
+customresourcedefinition.apiextensions.k8s.io/backupschedules.pingcap.com created
+customresourcedefinition.apiextensions.k8s.io/tidbmonitors.pingcap.com created
+customresourcedefinition.apiextensions.k8s.io/tidbinitializers.pingcap.com created
+customresourcedefinition.apiextensions.k8s.io/tidbclusterautoscalers.pingcap.com created
+```
+
+### 安装 TiDB Operator
+
+TiDB Operator 使用 Helm 安装，Helm 2 和 Helm 3 版本用法略有不同。可使用 `helm version --short` 命令查看安装的版本。
```

Comment:
It looks like Helm 2 and Helm 3's steps are almost the same except that users have to install tiller for Helm 2 first.

For the purpose of future maintenance, can you **reorganize this section** to merge the same steps?

Reply:
@dragonly Please address these comments~ Thanks.

Reply:
helm 2 and 3 will generate different output in step 3, which takes large page space.
Merging them may be a bad idea for reading fluency.